wgpu-hal/Cargo.toml

```diff
@@ -138,7 +138,7 @@ winapi = { version = "0.3", features = [
     "winuser",
     "dcomp",
 ] }
-d3d12 = { version = "0.7", features = ["libloading"], optional = true }
+d3d12 = { path = "../d3d12/", features = ["libloading"], optional = true }
```

Comment:
This should both be a path and a version I think, to let us publish the crate without changing this.

Reply:
Fixed up with 22d01e452.